packages/psss/src/store/auth.js

```diff
@@ -45,6 +46,22 @@ export const logout = () => ({
   type: LOGOUT,
 });
 
+export const updateProfile = (id, payload) => async (dispatch, getState, { api }) => {
+  try {
+    await api.put(`user/${id}`, null, payload);
+    const { body: user } = await api.get(`user/${id}`);
```

Comment:
```suggestion
    const { body: user } = await api.get(`me`);
```

Reply:
ah yes good catch. I missed adding the extra get me endpoint in the first PR